applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/cl_ref.dat

```diff
@@ -0,0 +1 @@
+ 0.327805503865
```

Comment:
Same.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668